src/modules/facility-activation-transaction/dto/create-facility-activation-transaction-request.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ENUMS } from '@ukef/constants';
+import { ValidatedFacilityIdentifierApiProperty } from '@ukef/decorators/validated-facility-identifier-api-property';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { UkefId } from '@ukef/helpers';
+
+export type CreateFacilityActivationTransactionRequest = CreateFacilityActivationTransactionRequestItem[];
+
+export class CreateFacilityActivationTransactionRequestItem {
+  @ValidatedFacilityIdentifierApiProperty({
+    description: 'The identifier of the facility to activate.',
+  })
+  readonly facilityIdentifier: UkefId;
+
+  @ValidatedNumberApiProperty({
+    description: 'In most situations value should be 3, it means auto approval.',
+    example: ENUMS.BUNDLE_STATUSES.SUBMIT_FOR_POSTING,
+    minimum: 0,
```

Comment:
Do we need `minimum: 0` if it needs to be a value from the enum anyway?

Reply:
I made attribute "minimum" optional, but removing it still causes issue in api-test helper, so I left this attribute in decorator.
I am creating TODO item now and maybe it can be done in another PR. Also happy todo it in this PR next week.

Reply:
Removed "minimum: 0" and created new test helper withNumberFieldValidationApiTests, it is copy of withNonNegativeNumberFieldValidationApiTests.